massa-grpc/src/stream/new_slot_execution_outputs.rs

```diff
@@ -240,75 +265,92 @@ fn get_filter(
                 },
                 grpc_api::new_slot_execution_outputs_filter::Filter::AsyncPoolChangesFilter(filter) => {
                     if let Some(filter) = filter.filter {
+                        let mut nested_filter = AsyncPoolChangesFilter::default();
```

Comment:
that will not work if i send a vec with [DestinationAddress, CanBeExecuted].

Reply:
Yep, we have update the current parent filter to Option<Vec<AsyncPoolChangesFilter>> which will be funny

Reply:
it's like new_endorsements https://github.com/massalabs/massa/blob/b9a93dc6f019652948d1787fd988a98d53026c65/massa-grpc/src/stream/new_endorsements.rs#L146

https://github.com/massalabs/massa/blob/b9a93dc6f019652948d1787fd988a98d53026c65/massa-grpc/src/stream/new_endorsements.rs#L174C37-L174C37